cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
@katyhuff 
I just attempted to run the xml one of the sample xml input in cycamore with xml inclusion with cyclus' native xml parser and it works!

For now, I will use the xinclude but if xml inclusion using ENTITY method instead?